.github/workflows/build_docker_image.yml

```diff
@@ -8,6 +8,9 @@ on:
       - main
   workflow_dispatch:
 
+env:
+  VTK_SHA: 6a898fd91da30d3ff903416bf856f0d1cea82cdf
```

Comment:
maybe a good time to update the vtk version to 1ab19927725cb4b86af8afaf255ed0ffc9fc913a ?